tests/testthat/helper-data.R

```diff
@@ -0,0 +1,14 @@
+# ------------------------------------------------------------------------------
+# for quantile regression tests
+
+library(modeldata)
+
+data("Sacramento")
+
+cols <- c("price", "beds", "baths", "sqft", "latitude", "longitude")
+Sacramento_small <- modeldata::Sacramento
+Sacramento_small$price <- log10(Sacramento_small$price)
+Sacramento_small <- Sacramento_small[, cols]
+
+sac_train <- Sacramento_small[-(1:5), ]
+sac_test  <- Sacramento_small[  1:5 , ]
```

Comment:
I don't see this being used either, can we remove the whole file?

Reply:
Sure. It'll stay in parsnip